modules/sdk-coin-sol/src/bigint-buffer-guard.ts

```diff
@@ -0,0 +1,28 @@
+try {
```

Comment:
can you add some docs here on why this file exists. It's not straightforward unless someone searches through the git blame.

Reply:
Right, but I meant like a comment in the code. 

Having important context outside the codebase makes the context hard to find (dig through a series of git blames).